### PR TITLE
mobilenet v2 passed

### PR DESCRIPTION
**Pull request by JiahaoYao**
- [ ] mobilenet-v2: tensorflow to caffe crash because of symmetric padding again
- [x] mobilenet-v2: tensorflow to mxnet/ pytorch/ coreml/ keras passed
- [ ] please test the cntk emitter

### CODE REVIEW

**Comment by kitstar** [comment]
TensorFlow mobilenet v2 to MXNet result:

error: 0.232831
L1 error: 49.62841
SNR: 21.360480764768198
PSNR: 39.348313358979546

**Comment by ujsyehao** [comment]
@JiahaoYao I notice the problem, tensorflow supports for asymmetric padding, but caffe(official version) just only support for symmetric padding, mmdnn **has the wrong handling** when dealing with asymmetric padding. Do you have some thoughts about solving the problem?
I plan to modify caffe to support for asymmetric padding, but it is related with mmdnn caffe_emitter, I need some help!

**Comment by JiahaoYao** [comment]
Cool ! I think one way to deal with the asymmetric problem is to classify two kinds of padding, `Top left` and `Bottom right`. Others might be to explicitly give the padding numbers in 4 dims, i.e. top, bottom, right and left.
